README.md

```diff
@@ -1,2 +1,13 @@
 # comments-profile
```

Comment:
Yay readme!!

Reply:
Wait I just ran your component locally, you should edit this to say to run "npm run seed" instead